priam/src/main/java/com/netflix/priam/tools/SimpleDBHelper.java

```diff
@@ -0,0 +1,35 @@
+package com.netflix.priam.tools;
+
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.netflix.priam.aws.SDBInstanceData;
+import com.netflix.priam.aws.auth.EC2RoleAssumptionCredential;
+import com.netflix.priam.aws.auth.ICredential;
+import com.netflix.priam.config.AmazonConfiguration;
+import com.netflix.priam.config.CassandraConfiguration;
+
+/*
+   Helper class to provide the SDBInstanceData object and can be extendable for any other purposes.
+ */
+public class SimpleDBHelper {
```

Comment:
I think this would work better as an injectable which itself is injected with the SDB credential provider.  Any time a non-trivial static method is used it makes unit testing more difficult.